extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+   * `INT32`
+   * `UINT32`
+   * `INT64`
+   * `UINT64`
+   * `FLOAT32`
+   * `FLOAT64`
+   * `BOOLEAN`
+   * `STRING`
+   * `ENUM`
+
+<!-- omit in toc -->
+#### class.property.componentCount
+
+The number of components per element for `ARRAY` elements.
+
+* **Type**: `integer`
+* **Required**: No
+* **Minimum**: ` >= 2`
+
+<!-- omit in toc -->
+#### class.property.normalized
+
+Specifies whether integer values are normalized. This applies both when `type` is an integer type, or when `type` is `ARRAY` with a `componentType` that is an integer type. For unsigned integer types, values are normalized between `[0.0, 1.0]`. For signed integer types, values are normalized between [-1.0, 1.0]. For all other types, this property is ignored.
+
+* **Type**: `boolean`
+* **Required**: No, default: `false`
+
+<!-- omit in toc -->
+#### class.property.max
+
+Maximum allowed values for property values. Only applicable for numeric types and fixed-length arrays of numeric types. For numeric types this is a single number. For fixed-length arrays this is an array with `componentCount` number of elements. `normalized` property has no effect on these values: they always correspond to the integer values.
+
+* **Type**: `number,array`
+* **Required**: No
+
+<!-- omit in toc -->
+#### class.property.min
+
+Minimum allowed values for property values. Only applicable for numeric types and fixed-length arrays of numeric types. For numeric types this is a single number. For fixed-length arrays this is an array with `componentCount` number of elements. `normalized` property has no effect on these values: they always correspond to the integer values.
+
+* **Type**: `number,array`
+* **Required**: No
+
+<!-- omit in toc -->
+#### class.property.default
+
+A default value to use when the property value is not defined. If used, `optional` must be set to true. The type of the default value must match the property definition: For `BOOLEAN` use `true` or `false`. For `STRING` use a JSON string. For numeric types use a JSON number. For `ENUM` use the enum `name`, not the integer value. For `ARRAY` use a JSON array containing values matching the `componentType`.
+
+* **Type**: `boolean,number,string,array`
+* **Required**: No
+
+<!-- omit in toc -->
+#### class.property.optional
+
+If true, this property is optional.
+
+* **Type**: `boolean`
+* **Required**: No, default: `false`
+
+<!-- omit in toc -->
+#### class.property.semantic
+
+An identifier that describes how this property should be interpreted. The semantic cannot be used by other properties in the class.
+
+* **Type**: `string`
+* **Required**: No
+* **Minimum Length**`: >= 1`
+
+<!-- omit in toc -->
+#### class.property.extensions
+
+* **Type**: `any`
+* **Required**: No
+
+<!-- omit in toc -->
+#### class.property.extras
+
+* **Type**: `any`
+* **Required**: No
+
+---------------------------------------
+<a name="reference-enum"></a>
+<!-- omit in toc -->
+### Enum
+
+An object defining the values of an enum.
```

Comment:
Not just the values, but the enum itself, which includes its values.  This creates the type.